src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -265,9 +267,11 @@ export const ContentMessageComponent = ({
           )}
         </div>
 
-        <div css={deliveredMessageIndicator}>
-          <DeliveredMessage isLastDeliveredMessage={isLastDeliveredMessage} />
-        </div>
+        {message.expectsReadConfirmation && (
+          <div css={deliveredMessageIndicator}>
+            <DeliveredMessage isLastDeliveredMessage={isLastDeliveredMessage} is1to1Conversation={is1to1} />
+          </div>
+        )}
```

Comment:
Wouldn't it be easier if we just didn't render the component at all if it's `!is1to1`? Passing a prop and rendering `null` doesn't really make sense to me 🤔  Same with `isLastDeliveredMessage`

Reply:
But we can render the div anyway and just render the message conditionally